package.json

```diff
@@ -68,7 +68,7 @@
     "hardhat-deploy": "^0.7.0-beta.9",
     "hardhat-gas-reporter": "^1.0.1",
     "inquirer": "8.0.0",
-    "p-queue": "^6.6.2",
+    "p-queue": "6.6.2",
```

Comment:
What's the reason for this change?

Reply:
@tmigone the p-queue suffered changes after major version 6, we could still have the major anchored to 6, I'll change it